cy-runner.yml

```diff
@@ -113,20 +109,21 @@ strategy:
     sendDashboard: true
     hardTries: 1
     stopOnFail: false
-    parallel: false
-    runInOrder: true
+    parallel: true
     specs:
       - cypress-shared/integration/b2b/2.2*
       - cypress-shared/integration/b2b/2.4*
+      - cypress-shared/integration/b2b/2.3*
     dependency:
       - cypress-shared/integration/b2b/2.1.1*
-  organizationAUsersTestCases:
+      - cypress-shared/integration/b2b/2.1.2*
+
+  orgA:
     enabled: false
     sendDashboard: true
     hardTries: 1
     stopOnFail: false
-    parallel: false
-    runInOrder: true
+    parallel: true
```

Comment:
This has to be in sequence

Reply:
Sorry, I didn't understand this one.

Reply:
We should run them in order
If we run them in parallel then it will fail